tests/phpunit/elementor/core/admin/test-admin.php

```diff
@@ -181,6 +181,8 @@ public function test_get_init_settings__returns_experiments_config() {
 				'title' => 'Experiment 1',
 				'state' => Experiments_Manager::STATE_ACTIVE,
 				'dependencies' => [],
+				'messages' => (object)[],
```

Comment:
add a case with messages, to see that's being passed properly

Reply:
Turned to an array as we discussed earlier.